topics/statistics/tutorials/genomic-llm-finetuning/tutorial.md

```diff
@@ -499,17 +503,27 @@ We will use the files `data_path` folder we just defined:
 >
 {: .question}
 
+
+Before we proceed we import some classes and functions from `scriptPython/function.py`:
+
+```python
+### LOAD FUNCTIONS MODULE
+import sys
+sys.path.append("scriptPython/")
+from functions import *
+```
+
 We use the `SupervisedDataset` class to load and prepare the datasets. This class handles the tokenization and formatting of the data, making it ready for model training and evaluation.
 
 ```python
 train_dataset = SupervisedDataset(
     tokenizer=tokenizer,
-    data_path=Path(data_path / "train.csv"),
+    data_path=os.path.join(data_path , "train.csv"),
```

Comment:
I would prefer to keep `pathlib` as it is more of a best practice. It is much easier and cleaner to work with, especially for cross-platform development.

Reply:
OK, although `os.path` is cross-platform too. 
Anyhow, the most important is then to use the correct syntax for `pathlib`:
`data_path=Path(data_path ) / "train.csv" ,`